src/Test/Utilities/Portable/TestResource.resx

```diff
@@ -961,7 +961,7 @@ L:
             Dim o4 = New MyObject(dynamic) With {.A = 0, .B = 0, .C = 0}
             Dim o5 = New With {Key .A = 0}
             Dim a() As Integer = {0, 1, 2, 3, 4, 5}
-            Dim a As (named As Integer, Integer) = (named:=1, 2)
+            Dim a As(named As Integer, Integer) =(named:=1, 2)
```

Comment:
Do you have a follow-up issue to sort this out? As we discussed yesterday, you shouldn't need this workaround for this test to pass.